### PR TITLE
Bugfix/is activity updates

### PR DESCRIPTION
**Pull request by jwong-d2l**

### CODE REVIEW

File: test/index.html [hunk 1]
```diff
@@ -22,14 +22,14 @@
 				'd2l-sequence-navigator-activity.html',
 				'd2l-sequence-navigator-item.html?wc-shadydom=true&wc-ce=true',
 				'd2l-sequence-navigator-item.html?dom=shadow&lazyRegister=true&useNativeCSSProperties=true',
-				'd2l-sequence-navigator-item.html?dom=shadow',
```
Comment (tinglisD2L) [inline test/index.html]: why not test the dom shadow case?

Comment (jwong-d2l) [reply test/index.html]: isn't that the default?

Comment (tinglisD2L) [reply test/index.html]: I'm not sure, but if it is, then the other `?dom=shadow` cases could be removed too

Comment (jwong-d2l) [reply test/index.html]: they probably could, but those tests I didn't really touch.

Comment (tinglisD2L) [reply test/index.html]: It looks like the default is shady DOM: https://www.polymer-project.org/1.0/docs/tools/tests#test-local-dom

Comment (jwong-d2l) [reply test/index.html]: that's a link to polymer 1.0 docs, the 2.0 docs seem to imply that the default is shadow (but they don't explicitly say)

**Pull request merged**